### PR TITLE
Update user-migration.mdx

### PR DESCRIPTION
**Pull request by Dkairu**
Please review verbiage and format to make sure it aligns with other docs.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by LizBaker** [comment]
gatsby pushed a fix yesterday so trying one more build

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 title: Migrate your users to the New Relic One user model 
-metaDescription: "For New Relic accounts with users on our original user model: how to migrate your users to our new model."
+metaDescription: "For New Relic organizations with users on our original user model: how to migrate your users to our new model."
 ---
 
-Starting April 12, 2021, we're allowing some customers who have users on our original user model to self-serve and migrate those users to be on the New Relic One user model.
+Starting April 12 of 2021, we're allowing some customers who have users on our original user model to self-serve and migrate those users to be on the New Relic One user model.
```
Comment (akristen) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:6]: Self-serve is hyphenated as a noun, but as a verb I believe Chicago manual drops the hyphen.

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 2]
```diff
@@ -141,17 +141,37 @@ Okta, Azure, and OneLogin have New Relic apps for both the original user model a
...
 
 When you reupload your list of users and complete this step, it will create user records on the New Relic One user model. In a later step, you’ll be able to transition these users' assets. 
 
-The new user records created for your users have the same login credentials: there's no need to reset passwords. If a user has a pending email verification status (pending being verified), that will also be transitioned over. 
-
-Tips and considerations: 
+Some tips and considerations: 
+* If using username/password for your users' login, they'll have the same login credentials on the new user record created: there's no need to reset passwords. If a user has a pending email verification status (pending being verified), that will also be transitioned over.
```
Comment (akristen) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:158]: The "If" sentence reads easier without the colon, then having "There's no need to reset passwords" be a punchier sentence. 

Also, might clarify at what point they will be transitioned over. Is there a time frame between verification and the transition? Is it immediate?

**Review by akristen — DISMISSED** [review]

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:6]: funny enough, I was just making a minor edit to retrigger the build. so sometimes when you see me making trivial edits it's just me trying to get the build to work ha

**Pull request merged**